### PR TITLE
Return policy name which allowed in enforce API

### PR DESCRIPTION
**Pull request by Pyav123**

### CODE REVIEW

File: src/coreEnforcer.ts [hunk 1]
```diff
@@ -485,7 +490,14 @@ export class CoreEnforcer {
       logPrint(reqStr);
     }
 
-    return res;
+    if(successReason){
+      response = { decision : res ,successRule : successReason };
```
Comment (Zxilly) [inline src/coreEnforcer.ts:494]: Maybe always return `[result, rule]` will be a better choice.

**Review by Zxilly — CHANGES_REQUESTED** [review]

File: src/coreEnforcer.ts (hunk 1 above)
Comment (Pyav123) [reply src/coreEnforcer.ts]: Done, please check

**Comment by hsluoyz** [comment]
@Pyav123 @Zxilly I think we already have the `enforceEx()` API to do this job. Why isn't it used? https://casbin.org/docs/en/management-api#enforceex

**Comment by Pyav123** [comment]
> @Pyav123 @Zxilly I think we already have the `enforceEx()` API to do this job. Why isn't it used? https://casbin.org/docs/en/management-api#enforceex

It is for php,go,pyton , need same for nodejs

**Comment by hsluoyz** [comment]
@Pyav123 fix all checks:

image

**Comment by hsluoyz** [comment]
Replaced by: https://github.com/casbin/node-casbin/pull/271

**Pull request closed without merging**